prod/westeurope/internal/api/functions_iopayportal/function_app/terragrunt.hcl

```
@@ -0,0 +1,98 @@
+dependency "subnet" {
+  config_path = "../subnet"
+}
+
+# External
+dependency "app_service_pagopaproxyprod" {
+  config_path = "../../../../external/pagopaproxyprod/app_service"
+}
+
+# Internal
+dependency "resource_group" {
+  config_path = "../../../resource_group"
+}
+
+dependency "subnet_apimapi" {
+  config_path = "../../../api/apim/subnet"
+}
+
+# Common
+dependency "virtual_network" {
+  config_path = "../../../../common/virtual_network"
+}
+
+dependency "application_insights" {
+  config_path = "../../../../common/application_insights"
+}
+
+dependency "key_vault" {
+  config_path = "../../../../common/key_vault"
+}
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_function_app?ref=v2.1.10"
+}
+
+inputs = {
+  name                = "iopayportalapi"
+  resource_group_name = dependency.resource_group.outputs.resource_name
+
+  app_service_plan_info = {
+    kind     = "elastic"
+    sku_tier = "ElasticPremium"
+    sku_size = "EP1"
+  }
+
+  runtime_version = "~3"
+
+  pre_warmed_instance_count = 2
+
+  application_insights_instrumentation_key = dependency.application_insights.outputs.instrumentation_key
+
+  app_settings = {
+    FUNCTIONS_WORKER_RUNTIME       = "node"
+    WEBSITE_NODE_DEFAULT_VERSION   = "10.14.1"
```

if will be this function
https://github.com/pagopa/io-pay-portal/blob/main/io-functions-pay-portal/.node-version
versions must be the same
```suggestion
    WEBSITE_NODE_DEFAULT_VERSION   = "12.19.1"
```
check also if `12.19.1` is a supported runtime version for azure functions. `12.18.0` is supported